components/Image/my-map-test.tsx

```diff
@@ -225,39 +240,87 @@ export default function MyMapTest({ source }: { source: string }) {
     setIsDragging(false)
     setIsFirstPress(true)
     setAccel(1)
+    setTouchDist(0)
     console.log('Mouse LEAVE Event function')
   }
+  const moveLeft = () => {
+    console.log(`Image scale for moveleft is: ${imgScale}`)
+    /*     let xDiff = -50/imgScale
+    console.log(`xdiff is : ${xDiff}`)
+    if (imgLeft + xDiff <= leftLimit) {
+      setImgLeft(leftLimit)
+    } else if (imgLeft + xDiff >= 0) {
+      setImgLeft(0)
+    } else setImgLeft(imgLeft + xDiff) */
+    setImgLeft(-2084)
+  }
+
+  const moveTop = () => {
+    let yDiff = -50 * imgScale
```

Comment:
Here is one of these numbers that can go into a constant's object above.

Reply:
This was a scratchpad component, this wont in the real component. Will be deleting component

Reply:
Done